src/constants.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Shared constants."""
```

Comment:
IMHO these should be defined in docs_directory.py instead. I consider this approach an anti-pattern and usually unjustified as it breaks encapsulation and leads to poor cohesion

Reply:
I agree, sometimes constants are required in a bunch of places which leads to circular imports that you can only resolve by putting certain constants into a different file. That is the reason for this, the first preference is to keep them with the module. I'll add a note with the reasoning